frontend/simple/js/state.js

```diff
@@ -148,16 +175,21 @@ const actions = {
         data: state[contractId]
       }))
     }
-    await db.saveSettings(settings)
+    await db.saveSettings(user, settings)
+    Vue.events.$emit('saved', user)
     console.log('saveSettings:', settings)
   },
   async loadSettings (
-    {commit, state}: {commit: Function, state: Object}
+    {commit, state}: {commit: Function, state: Object},
+    user: string
   ) {
-    const settings = await db.loadSettings()
-    console.log('loadSettings:', settings)
-    commit('setCurrentGroupId', settings.currentGroupId)
-    commit('setContracts', settings.contracts || [])
+    const settings = await db.loadSettings(user)
+    if (settings) {
+      console.log('loadSettings:', settings)
+      commit('setCurrentGroupId', settings.currentGroupId)
+      commit('setContracts', settings.contracts || [])
+    }
+    Vue.events.$emit('loaded', user)
```

Comment:
This is only waited for in one place, this file, and therefore it's not necessary because the place it's waited for can be `await`ed for.